### DISCUSSION TITLE
klog track from previous task to now

### DISCUSSION THREAD

**Discussion by QuAzI** [jotaen/klog#319] [Feature ideas & discussions]
Is there a way to use something like
```
klog track --append 'I did something...'
```
to track something from previous end point to NOW()

I would like to use this feature from git commit hook to automatically track finished features from commit message, but currently there is no way to get the last timestamp from klog.

**Comment by jotaen** [comment]
No, at this point there is no such functionality. The only feature that’s somewhat related is to use `klog switch`, but that doesn’t work in hindsight, but only if the previous entry is still open. E.g.:

```
2000-01-01
  15:00 - ? Some thing
```

Then doing `klog switch --summary 'Other thing'` at 15:34 would yield

```
2000-01-01
  15:00 - 15:34 Some thing
  15:34 - ? Other thing
```

**Reply by QuAzI** [reply]
That means there is no way to automate log with Git or some other software (like some CLI's for Jira, GitLab...).
User has to track everything by hand and at least twice (in both apps)
